src/main/resources/application.properties

```diff
@@ -50,6 +50,10 @@ spring.activemq.embedded-broker-name=embedded-broker
 spring.activemq.ws-transport-connector-url=ws://0.0.0.0:61624
 spring.activemq.stomp-transport-connector-url=stomp://0.0.0.0:61623
 
+spring.kafka.enabled=false
```

Comment:
What is this property used for? What will happen if I set the radio button to be Kafka mode but set this property to false here?

Reply:
This property was removed after discussion. (Perhaps we can handle issues in the console about its server disconnect warnings when registering a listener)